inference-engine/src/mkldnn_plugin/mkldnn_memory.cpp

```diff
@@ -304,6 +304,25 @@ memory::format MKLDNNMemory::GetPlainFormat(memory::dims dims) {
     }
 }
 
+memory::format MKLDNNMemory::GetPerChannelFormat(memory::dims dims) {
+    switch (dims.size()) {
+        case 0:
+            return memory::x;
```

Comment:
This line can be removed.

Reply:
Done in c1d47d09c49b216880a8b0b07848d33e7bb88372.